content/en/docs/tasks/inject-data-application/distribute-credentials-secure.md

```diff
@@ -43,13 +43,6 @@ username and password:
     kubectl create -f https://k8s.io/docs/tasks/inject-data-application/secret.yaml
     ```
 
-    {{< note >}}
```

Comment:
This change LGTM. The duplication was introduced by #9070

Reply:
I don't know why it got mixed up.
I removed the unrelated PR.